BraveWallet/Crypto/Stores/UserAssetsStore.swift

```diff
@@ -117,6 +122,27 @@ public class UserAssetsStore: ObservableObject {
       self?.updateSelectedAssets(network)
     }
   }
+  
+  func tokenInfo(by contractAddress: String, completion: @escaping (BraveWallet.BlockchainToken?) -> Void) {
+    timer?.invalidate()
+    timer = Timer.scheduledTimer(withTimeInterval: 0.25, repeats: false, block: { [weak self] _ in
```

Comment:
I think we only need a timer to debounce if we hit the network, local fetches should be instant